package/crossplane.yaml

```diff
@@ -2,6 +2,13 @@ apiVersion: meta.pkg.crossplane.io/v1alpha1
 kind: Provider
 metadata:
   name: provider-gcp
+  annotations:
+    company: Crossplane
+    maintainer: Nic Cope <negz@upbound.io>
```

Comment:
I wonder if we should setup some kind of group alias for this, rather than naming me specifically. @jbw976 are @crossplane.io emails a thing?

Reply:
yes, info@crossplane.io is a valid and monitored email address.  that could be reasonable to use here!

Reply:
There's also a CNCF mailing list specifically for crossplane maintainers that could work here too: cncf-crossplane-maintainers@lists.cncf.io.

info@crossplane.io may be cleanest though :)